test/integration/arbitrum-mainnet/Loans.fork.t.sol

```diff
@@ -103,33 +103,40 @@ abstract contract LoansTestBase is Test, DeploymentLoader {
 }
 
 contract LoansForkTest is LoansTestBase {
-    address cashAsset = 0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8; // USDC
-    address underlying = 0x82aF49447D8a07e3bd95BD0d56f35241523fBab1; // WETH
+    address constant cashAsset = 0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8; // USDC
```

Comment:
why? (just curious)

also, immutable should work this way too (but would have the correct casing)

Reply:
i first thought it may help with the stack too deep thing since i was battling with that throughout this whole thing (it didnt)
and then i just left it there cause thought it made sense syntatically since they dont change